### PR TITLE
Маячкин - Лабораторная работа #2

### PR DESCRIPTION
**Pull request by senyamac**

### CODE REVIEW

**Comment by senyamac** [comment]
@GerasimovDG
@MaksimenkoAN

**Comment by MaksimenkoAN** [comment]
👍

**Comment by GerasimovDG** [comment]
👍

**Comment by senyamac** [comment]
@kirill-korniakov 
@AntonGrishin 
@homksei

File: code/stack/viewmodel/tests_stack_viewmodel.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
+import unittest
+
+from stack.viewmodel.stack_viewmodel import StackViewModel
+
+
+class MyTestCase(unittest.TestCase):
```
Comment (AntonGrishin) [inline code/stack/viewmodel/tests_stack_viewmodel.py:6]: class TestStackViewModel

**Review by AntonGrishin — CHANGES_REQUESTED** [review]

File: code/stack/viewmodel/tests_stack_viewmodel.py (hunk 1 above)
Comment (senyamac) [reply code/stack/viewmodel/tests_stack_viewmodel.py]: Thank you. The name have been edited.

**Comment by senyamac** [comment]
@AntonGrishin

**Review by AntonGrishin — APPROVED** [review]

**Pull request merged**